Index: app/src/main/java/com/example/notesapp/Login.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.notesapp;\r\n\r\nimport androidx.appcompat.app.AppCompatActivity;\r\n\r\nimport android.os.Bundle;\r\n\r\npublic class Login extends AppCompatActivity {\r\n\r\n    @Override\r\n    protected void onCreate(Bundle savedInstanceState) {\r\n        super.onCreate(savedInstanceState);\r\n        setContentView(R.layout.activity_login);\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/notesapp/Login.java b/app/src/main/java/com/example/notesapp/Login.java
--- a/app/src/main/java/com/example/notesapp/Login.java	(revision ed27a3c76e8d5b533706e3a3d8fd5636ff358a5d)
+++ b/app/src/main/java/com/example/notesapp/Login.java	(date 1699586218098)
@@ -10,5 +10,7 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
+        getSupportActionBar().setTitle("Login Page");
     }
+
 }
\ No newline at end of file
Index: app/src/main/java/com/example/notesapp/Welcome.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.notesapp;\r\n\r\nimport androidx.appcompat.app.AppCompatActivity;\r\n\r\nimport android.os.Bundle;\r\n\r\npublic class Welcome extends AppCompatActivity {\r\n\r\n    @Override\r\n    protected void onCreate(Bundle savedInstanceState) {\r\n        super.onCreate(savedInstanceState);\r\n        setContentView(R.layout.activity_welcome);\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/notesapp/Welcome.java b/app/src/main/java/com/example/notesapp/Welcome.java
--- a/app/src/main/java/com/example/notesapp/Welcome.java	(revision ed27a3c76e8d5b533706e3a3d8fd5636ff358a5d)
+++ b/app/src/main/java/com/example/notesapp/Welcome.java	(date 1699586154067)
@@ -10,5 +10,6 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_welcome);
+        getSupportActionBar().setTitle("Welcome page");
     }
 }
\ No newline at end of file
